test/e2e/suite_test.go

```diff
@@ -60,8 +60,9 @@ func TestOperator(t *testing.T) {
 	t.Run("CreateMultiStargateAndDatacenter", e2eTest(ctx, "multi-stargate", true, createStargateAndDatacenter))
 	t.Run("CreateMultiDatacenterCluster", e2eTest(ctx, "multi-dc", false, createMultiDatacenterCluster))
 	t.Run("CheckStargateApisWithMultiDcCluster", e2eTest(ctx, "multi-dc-stargate", true, checkStargateApisWithMultiDcCluster))
-	t.Run("CreateSingleReaper", e2eTest(ctx, "single-dc-reaper", false, createSingleReaper))
-	t.Run("CreateMultiReaper", e2eTest(ctx, "multi-dc-reaper", false, createMultiReaper))
```

Comment:
It would be nice to have a test that deploys both Reaper and Stargate in the K8ssandraCluster.

Reply:
Added Stargate to CreateMultiReaper, let me know what you think. I had to modify cassandra-config.yaml slightly to make it compatible with C* 4.0. Obviously I could also change the test to use C* 3.11 but I think it's nice to have at least one test with C* 4.0 (it's the only one so far).

Reply:
LGTM